Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -5831,6 +5831,60 @@ public static void RedeemGooglePlayStorePurchaseForClass(string productId, strin
+        /// <param name="sandboxId">The Sandbox Id configured for the game making the purchase (this is the sandbox id from your epic online service configuration)</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        /// <param name="forPlayerWithUlid">Optional: Execute the request for the specified player. If not supplied, the default player will be used.</param>
+        public static void RedeemEpicStorePurchaseForPlayer(string accountId, string bearerToken, List<string> entitlementIds, string sandboxId, Action<LootLockerResponse> onComplete, string forPlayerWithUlid = null)
+        {
+            if (!CheckInitialized(false, forPlayerWithUlid))
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerResponse>(forPlayerWithUlid));
+                return;
+            }
+            var body = LootLockerJson.SerializeObject(new LootLockerRedeemEpicStorePurchaseForPlayerRequest()
+            {
+                account_id = accountId,
+                bearer_token = bearerToken,
+                entitlement_ids = entitlementIds,
+                sandbox_id = sandboxId
+            });
+            LootLockerServerRequest.CallAPI(forPlayerWithUlid, LootLockerEndPoints.redeemEpicStorePurchase.endPoint, LootLockerEndPoints.redeemEpicStorePurchase.httpMethod, body, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
+        }
+
+        /// <summary>
+        /// Redeem a purchase that was made successfully towards the Epic Store for a class that the current player owns
+        /// </summary>
+        /// <param name="accountId">The Epic account id of the account that this purchase was made for</param>
+        /// <param name="bearerToken">This is the token from epic used to allow the LootLocker backend to verify ownership of the specified entitlements. This is sometimes referred to as the Server Auth Ticket or Auth Token depending on your Epic integration.</param>
+        /// <param name="entitlementIds">The ids of the purchased entitlements that you wish to redeem</param>
+        /// <param name="sandboxId">The Sandbox Id configured for the game making the purchase (this is the sandbox id from your epic online service configuration)</param>
+        /// <param name="classId">The id of the class to redeem this purchase for</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        /// <param name="forPlayerWithUlid">Optional: Execute the request for the specified player. If not supplied, the default player will be used.</param>
+        public static void RedeemEpicStorePurchaseForClass(string accountId, string bearerToken, List<string> entitlementIds, string sandboxId, int classId, Action<LootLockerResponse> onComplete, string forPlayerWithUlid = null)
```

Comment:
Should class ID here be player class ID?
Class ID would be the one viewable in the console, player class ID would be the class belonging to this player.